### PR TITLE
Announcement panel

### PR DESCRIPTION
**Pull request by jakzaizzat**
**Changes**

- Add **What's new** tab
- Support redirect link 
- Tracked state based on [UserExperience](https://github.com/gallery-so/gallery/blob/main/schema.graphql#L621)

**Notes**
- Need to merge the backend PR first - https://github.com/gallery-so/go-gallery/pull/881

**Demo**

https://user-images.githubusercontent.com/4480258/227838522-edd8d566-9adb-4db9-b945-d8680f4b6e44.mp4

https://user-images.githubusercontent.com/4480258/228129287-88fb22a3-be2c-4984-8c1e-2e001c5ab5f2.mp4

### CODE REVIEW

File: apps/web/src/components/Announcement/constants.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    description: 'We will be performing maintenance on the site on Feb 2023.',
+    date: '2023-03-14T04:56:26.154845Z',
+  },
+  {
+    key: 'TwitterConnectionOnboardingUpsell',
+    title: 'Connect Twitter',
+    description: 'Connect your Twitter account to your profile.',
+    date: '2023-03-14T04:56:26.154845Z',
+  },
+];
```
Comment (jakzaizzat) [inline apps/web/src/components/Announcement/constants.ts:29]: This is just dummy content. Lmk what kind of content should we put first

File: apps/web/src/components/Announcement/AnnouncementList.tsx [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+import colors from '../core/colors';
+import { HStack, VStack } from '../core/Spacer/Stack';
+import { BaseM, BaseS } from '../core/Text/Text';
+import useAnnoucement, { AnnouncementType } from './useAnnouncement';
+
+type Props = {
+  queryRef: AnnouncementListFragment$key;
+};
+
+export default function AnnouncementList({ queryRef }: Props) {
```
Comment (Robinnnnn) [inline apps/web/src/components/Announcement/AnnouncementList.tsx:19]: so clean 😍

File: apps/web/src/components/Announcement/constants.ts (hunk 1 above)
Comment (Robinnnnn) [reply apps/web/src/components/Announcement/constants.ts:29]: let's make a new experience flag for mementos, and remove the rest. will need to add the experience to the backend- here's the [loom video](https://www.loom.com/share/97a3402a018941a491410278fdb789f7) for doing that

```js
export const ANNOUNCEMENT_CONTENT = [
  {
    key: 'UpsellMintMemento4',
    title: 'Now Minting: Blooming Connections',
    description: 'Gallery Memento #4 is now available for minting, a beautiful and symbolic representation of the growing network within the Gallery community.',
    date: '2023-03-27T13:00:00.154845Z',
    link: '/mint/mementos',
  },
];
```

File: apps/web/src/components/Announcement/useAnnouncement.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+  key: UserExperienceType;
+  title: string;
+  description: string;
+  date: string;
+  time: string | null; // time since date
+  experienced: boolean;
+  link?: Route;
+};
+
+export default function useAnnoucement(queryRef: useAnnouncementFragment$key) {
```
Comment (Robinnnnn) [inline apps/web/src/components/Announcement/useAnnouncement.tsx:23]: typo: `Announcement`

**Review by Robinnnnn — COMMENTED** [review]
@jakzaizzat this looks soooo sick!!!

let's make sure there's a notification dot here when there are unread announcements
Screenshot 2023-03-27 at 2 32 20 PM

**Comment by jakzaizzat** [comment]
@Robinnnnn The mobile view looks fine!

https://user-images.githubusercontent.com/4480258/228120003-faeac105-6c35-4228-be42-20f6c2f4d781.mp4

**Pull request merged**